lib/beaker/logger.rb

```diff
@@ -83,6 +83,25 @@ def initialize(*args)
 
       @destinations = []
 
+      @color_levels = options[:color_levels] || {
+        :error => RED,
+        :warn => BRIGHT_RED,
+        :success => MAGENTA,
+        :notify => BLUE,
+        :info => GREEN,
+        :debug => WHITE,
+        :trace => BRIGHT_YELLOW,
+        :perf => BRIGHT_MAGENTA,
+        :host => YELLOW
+      }
+
+      # Jenkins exposed variable - should be present on the slave directing
+      # the beaker run
+      ci_build = ENV['BUILD_NUMBER'] != nil
+
+      @color_levels[:notify] = NORMAL if ci_build
```

Comment:
I'm not sure I understand the need for 2 options for this - I think you should trust that if a user has overridden the color values, they know what they're doing.  Such changes could be part of local settings like `Gemfile.local` that aren't checked into source control.

FWIW, having Beaker ignore user specified values in an effort to "do what it thinks is best" has been a major source of headaches in the past (see `:type`).